pkg/controller/compliancescan/scantype.go

```diff
@@ -239,6 +240,14 @@ func (nh *nodeScanTypeHandler) gatherResults() (compv1alpha1.ComplianceScanStatu
 
 		// we output the last result if it was an error
 		if result == compv1alpha1.ResultError {
+			if !nh.getScan().IsStrictNodeScan() {
+				errCode := foundCM.Data["exit-code"]
+				// If the pod was unschedulable and the scan is not
+				// strict, we can skip the error
+				if errCode == common.PodUnschedulableExitCode {
+					continue
```

Comment:
It might be helpful to have a warning that the result was skipped under the non-strict mode.